credentials/apps/credentials/templatetags/file.py

```
@@ -0,0 +1,50 @@
+"""
+Template tags and helper functions for escaping script tag.
+"""
+from django import template
+from django.conf import settings
+from django.template.loader import select_template
+from django.utils.translation import get_language
+register = template.Library()
+
+
+@register.filter(name='translate_svg_path')
+def translate_svg_path(filepath):
+    """
+    Uses the filepath parameter to construct a set of potential language suffixed file names, and return
+    the template_name of the first template matching the current language, default language if otherwise,
+    or if neither the base path requested. Throws a TemplateNotFound exception if none are found.
+
+    The format necessary for filepath should match 'some/path/filename.svg'
+    This will search for 'some/path/filename-**.svg' where ** is the requested/default language tags.
+    """
+    paths = construct_svg_language_paths(filepath, get_language())
+    return select_template(paths).origin.template_name
+
+
+def construct_svg_language_paths(filepath, language, default=settings.LANGUAGE_CODE):
+    """
+    Creates an array containing between two and five strings a guaranteed in order:
+        1. The requested language added before the svg suffix (e.g. filepath-es-419.svg)
+        1. (optional) The requested language two character code added before the svg suffix (e.g. filepath-es.svg)
+        1. (optional) The default language added before the svg suffix (e.g. filepath-en-US.svg)
+        1. (optional) The default language two character code added before the svg suffix (e.g. filepath-en.svg)
+        1. The unmodified requested filepath (e.g. filepath.svg)
+    """
+    paths = [filepath.replace('.svg', '-' + language + '.svg')]
```

What if this is called with a file path that doesn't end in `.svg`? We should probably raise an error in that case if it is a hard requirement for this API.

---

Great suggestion :+1: will add that in